implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/v2/DynatraceExporterV2Test.java

```diff
@@ -346,6 +343,110 @@ void toLongTaskTimerLine() {
                         + clock.wallTime());
     }
 
+    private static void busyWait(LongTaskTimer ltt) {
+        ltt.record(() -> {
+            while (!Thread.currentThread().isInterrupted()) {
+                try {
+                    Thread.sleep(200);
+                }
+                catch (InterruptedException ignored) {
+                }
+            }
+        });
+    }
+
+    @Test
+    void longTaskTimerWithSingleValueExportsConsistentData() throws InterruptedException {
+        // In the past, there were problems with the LongTaskTimer: In cases where only
```

Comment:
Maybe would be good to have the name of the metric that you found to be the culprit? Maybe mention it here or in the PR so we have some "history of why"